package.json

```diff
@@ -98,4 +98,4 @@
     "test": "jest",
     "test:coverage": "jest --coverage"
   }
-}
+}
```

Comment:
Not sure why the newline was removed here

Reply:
My prettier parser... I'll put it back.